app/src/main/AndroidManifest.xml

```diff
@@ -36,7 +36,8 @@
             android:name=".circle_of_trust.Trustees"
             android:label="@string/title_activity_trustees"
             android:parentActivityName=".MainActivity"
-            android:windowSoftInputMode="stateHidden">
+            android:windowSoftInputMode="adjustPan"
```

Comment:
Sorry mam....I was testing it with something else and forgot to change it back

Its supposed to be **android:windowSoftInputMode="adjustPan|stateAlwaysHidden"**

should I change my commit?